### DISCUSSION TITLE
easyocr doesn't response after Reader and readtext

### DISCUSSION THREAD

**Discussion by DonutPPP** [JaidedAI/EasyOCR#1118] [Q&A]
I just try to user easyocr. But after I run code I found noting response in terminal VS code.
I don't understand why it don't have anything response please advise me.
 
![python](https://github.com/JaidedAI/EasyOCR/assets/88708961/d08921f9-6e32-4d3a-a098-5f9f8fb1723b)

**Comment by murgado** [comment]
Hey @DonutPPP ! I had the same problem and just fixed it downgrading easyocr to 1.6.2 and it now works fine, hope this alsou helps you.

**Reply by DonutPPP** [reply]
@murgado Thank you very much. I will try your solution.

**Reply by Qamar2315** [reply]
Is this issue solved?
